org.eclipse.scanning.event/src/org/eclipse/scanning/event/remote/_AbstractRemoteDevice.java

```diff
@@ -39,12 +39,14 @@ private _AbstractRemoteDevice(URI uri, IEventService eservice) throws EventExcep
 	@SuppressWarnings("unchecked")
 	_AbstractRemoteDevice(DeviceRequest req, URI uri, IEventService eservice) throws EventException, InterruptedException {
 		this(uri, eservice);
+	    logger.info("Setting "+getClass().getSimpleName()+" to timeout " + RemoteServiceFactory.getTime() + " "+RemoteServiceFactory.getTimeUnit());

```

Comment:
Do we need getClass().getSimpleName() here? The logger adds the class name

2017-04-11 15:51:57,199 INFO  org.eclipse.scanning.event.remote._RunnableDeviceService - Setting _RunnableDeviceService to timeout 500 ms